core/CronArchive.php

```diff
@@ -740,7 +749,7 @@ private function logNetworkError($url, $response)
     private function checkResponse($response, $url)
     {
         if (empty($response)
-            || stripos($response, 'error') !== false
+            || preg_match('/(?<!\/\* WP IGNORE |[a-zA-Z])(error)(?![a-zA-Z])/i', $response)
```

Comment:
This makes the search less inclusive. Instead of counting any occurrence of `error`, whether it's part of a larger word or not, this only counts the word `error`. For example, it no longer counts the `errorHandler` class which was in the trace from some warnings. This also ignores the `WP IGNORE ERROR` comment in the SQL statement from core/DataAccess/LogAggregator.php.

Reply:
Same as on the other change. An API response should always only include the response itself and no errors or error stacks. So I think this shouldn't be needed if the error handling would be fixed.

Reply:
True. If the API only returned a JSON response as is normally expected, this wouldn't be necessary. That said, without more background into why the API sometimes returns log messages with the response and this code that's been written expecting that, I  was just trying to make this check less inclusive in order to reduce the number of false positives in the error check.